blog/2020-03-01-CombiningBPMNWithOptaplanner.adoc

```diff
@@ -0,0 +1,611 @@
+= Combining BPMN with Optaplanner
+:page-interpolate: true
+:awestruct-author: Christopher-Chianelli
+:awestruct-layout: blogPostBase
+:awestruct-tags: [useCase]
+:awestruct-date: 2020-03-01
+
+Many Business Processes modelled with BPMN diagrams. When the Business
+Process involves optimization at some step,
+https://www.optaplanner.org/[OptaPlanner] can perform that task for us. Now with
+https://kogito.kie.org/[Kogito], it is easier then ever before to integrate
+OptaPlanner into your BPMN diagrams! We will show how simple it is by creating a
+Business Process for scheduling a flight in an airline from scratch using
+OptaPlanner, BPMN and Spring Boot in 15 minutes.
+
+== Project Setup
+
+Create a new folder to hold this project. Run the following command to
+create a new Kogito Spring Boot project:
+
+```
+mvn archetype:generate \
+-DarchetypeGroupId=org.kie.kogito \
+-DarchetypeArtifactId=kogito-springboot-archetype \
+-DarchetypeVersion=8.0.0-SNAPSHOT \
```

Comment:
Build fails:
>The desired archetype does not exist (org.kie.kogito:kogito-springboot-archetype:8.0.0-SNAPSHOT)

Reply:
OK, I needed to build the whole `kogito-runtimes` repository with Java 11. This needs to be mentioned as a prerequisite.

Reply:
Changed to quarkus project creator, doesn't have any of the MANY kogito archetype issues.